packages/canvas-panel/src/helpers/eventbus.ts

```diff
@@ -0,0 +1,65 @@
+import { ChoiceDescription } from 'react-iiif-vault/*';
```

Comment:
```suggestion
import { ChoiceDescription } from 'react-iiif-vault';
```

(bug in RIV with imports)